DataManagementSystem/DB/FileCatalogComponents/WithFkAndPs/DirectoryClosure.py

```diff
@@ -522,33 +522,6 @@ def _setDirectoryParameter( self, path, pname, pvalue ):
     else:
       return DirectoryTreeBase._setDirectoryParameter( self, path, pname, pvalue )
 
-
-
-  def setDirectoryGroup( self, path, gname ):
```

Comment:
Those methods are untouched ( only _ prepended ) but moved to DirectoryTreeBase which DirectoryClosure inherits. If you need your own implementation, add it here